discord_artifact.py

```diff
@@ -0,0 +1,13 @@
+import dataclasses
+from dataclasses import dataclass, field
+
+
+@dataclass
+class Artifact:
+    container_id: str
+    name: str
+    cef: dict = field(default_factory=lambda: {"URL": "", "Type": "", "Description": ""})
+
+
+def artifact_to_dict(artifact: Artifact):
+    return dataclasses.asdict(artifact)
```

Comment:
not sure if there's a real need to introduce function that does the same as other function just to... rename it. It's "almost" the same as:

```python
artifact_to_dict = dataclasses.asdict
```

turn on auto new-lines at the end of the files please :)